qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/QuteSupportForTemplate.java

```diff
@@ -368,6 +368,9 @@ private static boolean isIterable(IType iterableType, IProgressMonitor monitor)
 	}
 
 	private static boolean isValidField(IField field) throws JavaModelException {
+		if (field.isRecordComponent()) {
+			return true;
+		}

```

Comment:
not necessary, given the previous comment

Reply:
fixed